src/token/CustomERC721.sol

```diff
@@ -432,7 +432,7 @@ contract CustomERC721 is NonReentrant, ERC721H, LazyMint, DelayedReveal, ICustom
     uint256 _amount,
     string calldata _baseURIForTokens,
     bytes calldata _data
-  ) public override onlyOwner returns (uint256 batchId) {
+  ) public override returns (uint256 batchId) {
```

Comment:
We should not remove `onlyOwner` from this. It needs to only be callable by the deployer of the contract

Reply:
Yes of course, It is mandatory, I don't know why it is removed